src/features/courses/course-controller.ts

```diff
@@ -2455,7 +2457,7 @@ class CourseController {
                         if (_.isNil(question.grades)) throw new RederlyExtendedError('Impossible! Found an assessment question without a grade.');
                         const version = await courseRepository.getCurrentInstanceForQuestion({questionId: question.id, userId});
                         question.webworkQuestionPath = version?.webworkQuestionPath ?? question.webworkQuestionPath;
-                        question.grades.randomSeed = version?.randomSeed ?? question.grades.randomSeed;
+                        question.grades[0].randomSeed = version?.randomSeed ?? question.grades[0].randomSeed;
```

Comment:
Do we need an array length check here?

Reply:
grades is an include -- if it's included at all, this will suffice? I think sequelize won't include an empty array here if there's no match.

Reply:
It will include an empty array if there is no match